src/pages/commits/CommitsWrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { ApolloError } from "@apollo/client";
+import styled from "@emotion/styled";
+import { Skeleton } from "antd";
+import { PageWrapper } from "components/styles";
+import { MainlineCommitsQuery } from "gql/generated/types";
+
+export const CommitsWrapper: React.FC<{
+  versions: MainlineCommitsQuery["mainlineCommits"]["versions"];
+  error?: ApolloError;
+  chartType: "percentage" | "absolute";
+  isLoading: boolean;
+}> = ({ versions, isLoading, error }) => {
+  if (error) {
+    return <PageWrapper>ERROR</PageWrapper>;
+  }
+  if (isLoading) {
+    return <StyledSkeleton active title={false} paragraph={{ rows: 6 }} />;
+  }
+  if (!isLoading && versions?.length !== 0) {
+    return <FlexRowContainer />;
```

Comment:
Can we just render a few empty `<CommitChart />` elements  here with version descriptions  under them as placeholders?